common/src/main/java/com/wynntils/wc/custom/item/properties/HorseProperty.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wc.custom.item.properties;
+
+import com.wynntils.mc.utils.ItemUtils;
+import com.wynntils.wc.custom.item.WynnItemStack;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class HorseProperty extends ItemProperty {
+
+    private final Pattern HORSE_PATTERN = Pattern.compile(
+            "§7Tier (\\d)§6Speed: (\\d+)/(\\d+)§6Jump: \\d+/\\d+§5Armour: None§bXp: (\\d+)/100(?:§cUntradable Item)?(:?§7Name: (.+))?");
+    private int xp = -1;
+    private int level = -1;
+    private int maxLevel = -1;
+    private int tier = -1;
+    private String name = null;
```

Comment:
empty string as default value?

Reply:
Yes, agree. Strings should be "", not null, if at all possible.